students/BRACH_Jakub/lesson_01_basics/L01P01_Three_Numbers.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python3
+number_of_factors = 3
+summ = 0
+
+for x in range(0, number_of_factors):
+   import math
+   print('Enter the number {0:d} of {1:d}:'.format(x, number_of_factors))
+   summ = summ + input()
+
+print ("The summ is:{0:d}".format(summ))
```

Comment:
it's okay, but could be simplified to `'{:d}'` or just `{}`

Reply:
OK.